cosmos/ununifi-beta-v1.json

```diff
@@ -0,0 +1,55 @@
+{
+  "rpc": "http://a.lcd.ununifi.cauchye.net:26657",
+  "rest": "https://a.lcd.ununifi.cauchye.net:1318",
+  "nodeProvider": {
+    "name": "UnUniFi",
+    "email": "info@ununifi.io",
+    "website":"https://ununifi.io/"
+  },
+  "chainId": "ununifi-beta-v1",
+  "chainName": "UnUniFi",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/ununifi/chain.png",
```

Comment:
@Senna46 
Link will be changed from `ununifi/chain.png` to `ununifi-beta-v1/chain.png`